src/python/WMCore/WMRuntime/Startup.py

```diff
@@ -36,6 +37,15 @@
     Bootstrap.createInitialReport(job=job, reportName=reportName)
     monitor = Bootstrap.setupMonitoring(logName=reportName)
 
+    logging.info("Creating WM runtime information json")
+    jsonPath, jobType = Bootstrap.createWMRuntimeJson(os.getcwd())
+
+    if jobType in ("Production", "Processing"):
```

Comment:
To avoid loading this json file from the filesystem, how about moving this logic under `createWMRuntimeJson`? Another option would be to return the python dictionary and print it here - instead of loading the json file.